doc/getting_started/getting_started.rst

```diff
@@ -21,11 +32,28 @@ using Git to clone the repository anonymously. Enter:
 .. code-block:: console
 
    $ cd ~
-   $ git clone https://github.com/zephyrproject-rtos/zephyr.git
+   > cd %userprofile%
```

Comment:
Don't like these intermingled OS instructions. Just make two code blocks, one for Windows and the other for Linux/macOS